Amazon.QLDB.Driver.Serialization/Document.cs

```diff
@@ -0,0 +1,7 @@
+namespace Amazon.QLDB.Driver.Serialization
+{
+    public class Document
```

Comment:
Where does this get used?

Reply:
Anyone inserting/updating rows in QLDB. i.e. not in the serializer or driver code, but consuming libraries.